src/pages/workspace/accounting/WorkspaceAccountingPage.tsx

```diff
@@ -0,0 +1,238 @@
+import React, {useCallback, useMemo, useRef, useState} from 'react';
+import {ActivityIndicator, View} from 'react-native';
+// import ConnectToQuickbooksOnlineButton from './qboConnectionButton';
+import Button from '@components/Button';
+import ConfirmModal from '@components/ConfirmModal';
+import HeaderWithBackButton from '@components/HeaderWithBackButton';
+import * as Expensicons from '@components/Icon/Expensicons';
+import * as Illustrations from '@components/Icon/Illustrations';
+import type {MenuItemProps} from '@components/MenuItem';
+import MenuItemList from '@components/MenuItemList';
+import ScreenWrapper from '@components/ScreenWrapper';
+import ScrollView from '@components/ScrollView';
+import Section from '@components/Section';
+import ThreeDotsMenu from '@components/ThreeDotsMenu';
+import type ThreeDotsMenuProps from '@components/ThreeDotsMenu/types';
+import useLocalize from '@hooks/useLocalize';
+import useTheme from '@hooks/useTheme';
+import useThemeStyles from '@hooks/useThemeStyles';
+// import useWaitForNavigation from '@hooks/useWaitForNavigation';
+import useWindowDimensions from '@hooks/useWindowDimensions';
+import AdminPolicyAccessOrNotFoundWrapper from '@pages/workspace/AdminPolicyAccessOrNotFoundWrapper';
+import FeatureEnabledAccessOrNotFoundWrapper from '@pages/workspace/FeatureEnabledAccessOrNotFoundWrapper';
+import PaidPolicyAccessOrNotFoundWrapper from '@pages/workspace/PaidPolicyAccessOrNotFoundWrapper';
+import withPolicy from '@pages/workspace/withPolicy';
+import type {WithPolicyProps} from '@pages/workspace/withPolicy';
+import type {AnchorPosition} from '@styles/index';
+import CONST from '@src/CONST';
+
+function WorkspaceAccountingPage({policy}: WithPolicyProps) {
```

Comment:
It was agreed on the design doc that we would call this `PolicyAccountingPage`. According to the naming agreement on [here](https://expensify.slack.com/archives/C03TQ48KC/p1709066269962319?thread_ts=1706792648.640989&cid=C03TQ48KC), we should be using `Policy` instead of `Workspace` for referring to policies in text that is not user facing.

cc @hayata-suenaga

Reply:
I can create a quick PR to fix the issue thank you for noticing it

Reply:
No worries, I'm taking care of it in the PR where we add Authentication

Reply:
thank you!